src/ray/core_worker/actor_handle.h

```diff
@@ -70,6 +70,10 @@ class ActorHandle {
 
   int64_t MaxTaskRetries() const { return inner_.max_task_retries(); }
 
+  bool IsPersistedToGCS() const { return is_persisted_to_gcs_; }
```

Comment:
I think this also needs to acquire the mutex! For some reason, the thread annotations don't catch these bad accesses when the method is implemented in the .h file.

Reply:
Thanks for catching this!